intro.rst

```diff
@@ -0,0 +1,16 @@
+Introduction
+============
+
+Behaviour Driven Development
+----------------------------
+
+Once you're up and running with Behat, you can learn more about behaviour
+driven development via the following links. Though both tutorials are specific
+to Cucumber, Behat shares a lot with Cucumber and the philosophies are one
+and the same.
```

Comment:
Ok, I've added a "Next" and "Previous" button just as it works in [phpspec](http://phpspec.readthedocs.org/en/latest/manual/introduction.html). I've updated RTFD to make sure that we can track and visualize all the changes:

[Behat v3.0 index page](http://behat-docs.readthedocs.org/en/v3.0/)
[Behat v3.0 introduction page](http://behat-docs.readthedocs.org/en/v3.0/intro.html)